client/src/webview/SampleView.ts

```diff
@@ -70,6 +72,7 @@ export class SampleView extends View {
 
                 // Refresh the webviews html
                 panel.webview.html = await this.getHTMLString();
+                Reporter.instance().sendTelemetryEvent('Sample Opened', {sample: this.sampleName, language: language});
```

Comment:
I think we'll want to send the `fileName` (FabCar Contract) and `fileType` (contracts/applications) as well